src/components/Main/Main.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem';
+
+type Props = {
+  todos: Todo[],
+  tempTodo: Todo | null,
+  isTempLoading: boolean,
+  setTodos: (todos: Todo[]) => void,
+  pushError: (title: string) => void,
+  toBeCleared: Todo[] | undefined,
+};
+
+export const Main: React.FC<Props> = ({
+  todos,
+  tempTodo,
+  isTempLoading,
+  pushError,
+  setTodos,
+  toBeCleared,
+}) => {
+  return (
+    <section className="todoapp__main">
+      {todos.map(todo => (
+        <TodoItem
+          key={todo.id}
+          todo={todo}
+          todos={todos}
+          pushError={pushError}
+          setTodos={setTodos}
```

Comment:
Pass the 'isTempLoading' prop to the 'TodoItem' component when 'tempTodo' is not null.

Reply:
<img width="576" alt="image" src="https://github.com/mate-academy/react_todo-app-add-and-delete/assets/17066980/a302f63f-e801-48c8-a8a9-a75bb5ac6595">